### PR TITLE
UIREQ-794: Сannot add tags to record in the Requests app

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
A closed Request should not be edited. Hence disable the tags Icon Button on closed request.
Issue: https://issues.folio.org/browse/UIREQ-794

## Screenshots

Closed Request - Tags Icon Button disabled
image

Open Request - Tags Icon Button remain enabled
image

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by artem-blazhko — APPROVED** [review]

File: src/ViewRequest.js [hunk 1]
```diff
@@ -408,6 +408,9 @@ class ViewRequest extends React.Component {
 
     const tags = ((request && request.tags) || {}).tagList || [];
 
+    const requestStatus = get(request, ['status'], '-');
+    const isRequestClosed = requestStatus.startsWith('Closed');
```
Comment (Dmitriy-Litvinenko) [inline src/ViewRequest.js:412]: Hello, @Terala-Priyanka thank you. Could you please investigate possible of use closed request status from https://github.com/folio-org/ui-requests/blob/master/src/constants.js#L32 for comparison?
CC: @zburke , @artem-blazhko

Comment (zburke) [reply src/ViewRequest.js:412]: Sure, converting that to a function/list of statuses is a good idea to make sure open and closed are handled consistently across the application. I've advocated for a while (like, years) that the `status` attribute should itself have a boolean `isOpen` attribute to make these kinds of filters easier. Alas.

Comment (Terala-Priyanka) [reply src/ViewRequest.js]: @Dmitriy-Litvinenko  
Thank you for the observation. I have updated the code.

**Review by zburke — APPROVED** [review]
Still LGTM

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**